Src/PerformanceCollector/NetCore/Implementation/QuickPulse/QuickPulseServiceClient.cs

```diff
@@ -0,0 +1,372 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
```

Comment:
QuickPulseServiceClient.cs looks extremely similar for netcore and netfull implementation. Can you keep it in shared and do compilation flags to avoid code duplication?